docs/serving/samples/hello-world/helloworld-go/index.md

```diff
@@ -5,4 +5,231 @@ weight: 1
 type: "docs"
 ---
 
-{{% readfile file="README.md" %}}
+This guide describes the steps required to to create the `helloworld-go` sample app
+and deploy it to your cluster.
+The sample app reads a `TARGET` environment variable, and prints `Hello ${TARGET}!`.
+If `TARGET` is not specified, `World` is used as the default value.
+
+## Prerequisites
+
+You will need:
+- A Kubernetes cluster with [Knative installed and DNS configured](../../../../install/README.md).
+- [Docker](https://www.docker.com) installed and running on your local machine, and a Docker Hub account configured.
+- (optional) The Knatice CLI client [kn](https://github.com/knative/client/releases) that simplifies the deployment. Alternative you can also use `kubectl` and apply resource files directly.
+
+## Build
```

Comment:
```suggestion
## Building
```

We use deploying further down so I'd make them all gerunds or none of them

Reply:
Same, fixed both.